lib/livebook/storage/ets.ex

```diff
@@ -48,14 +46,19 @@ defmodule Livebook.Storage.Ets do
 
   @impl Livebook.Storage
   def fetch_key(namespace, entity_id, key) do
-    @table_name
+    :persistent_term.get(__MODULE__)
     |> :ets.match({{namespace, entity_id}, key, :"$1", :_})
     |> case do
       [[value]] -> {:ok, value}
       [] -> :error
     end
   end
 
+  @spec config_file_path() :: Path.t()
+  def config_file_path() do
+    Path.join([Livebook.Config.data_path(), "livebook.#{Mix.env()}.ets"])
```

Comment:
As of #1003 the `data_path` will already be env-specific

```suggestion
    Path.join([Livebook.Config.data_path(), "livebook.ets"])
```